test/test_helper.rb

```diff
@@ -7,4 +7,5 @@
 require "./lib/games"
 require "./lib/game_teams"
 require "./lib/teams"
+require "./lib/games_statistics"
```

Comment:
Thanks for the reminder, I had to add mine to the test_helper also.  Question, what is your test coverage percentage, because mine has dropped really low, and I don't know if it's because of the module.  Are we supposed to test the module?